public/setting.html

```diff
@@ -0,0 +1,50 @@
+<html>
+  <head>
+    <title>タイピングゲーム</title>
+    <link rel="stylesheet" href="./stylesheets/setting.css" />
+
+    <script type="module" src="./javascripts/main.js"></script>
+  </head>
+
+  <body>
+    <h2>設定</h2>
+    <p>Volume</p>
+    <div>
+      <input
+        type="range"
+        id="Volume"
+        name="Volume"
+        min="0"
+        max="100"
+        value="90"
+        step="2"
+      />
+    </div>
+
+    <p>BGM</p>
+    <div>
+      <input
+        type="range"
+        id="SE"
+        name="BGM"
+        min="0"
+        max="100"
+        value="90"
+        step="2"
+      />
+    </div>
+
+    <p>SE</p>
+    <div>
+      <input
+        type="range"
+        id="volume"
+        name="volume"
+        min="0"
+        max="100"
+        value="90"
+        step="2"
+      />
```

Comment:
以下のsuggestionを取り込んでください。

```suggestion
<div data-screen="setting">
      <h2>設定</h2>
      <div class="volume-control">
        <p>Volume</p>
        <input
          type="range"
          id="volume"
          name="volume"
          min="0"
          max="100"
          value="90"
          step="2"
        />
      </div>
      <div class="volume-control">
        <p>BGM</p>
        <input
          type="range"
          id="BGM"
          name="BGM"
          min="0"
          max="100"
          value="90"
          step="2"
        />
      </div>
      <div class="volume-control">
        <p>SE</p>
        <input
          type="range"
          id="SE"
          name="SE"
          min="0"
          max="100"
          value="90"
          step="2"
        />
      </div>
```